lib/routes/_core_profile.js

```diff
@@ -27,7 +27,8 @@ module.exports = {
       email: Joi.string().optional(),
       locale: Joi.string().optional(),
       amrValues: Joi.array().items(Joi.string().required()).optional(),
-      twoFactorAuthentication: Joi.boolean().optional()
+      twoFactorAuthentication: Joi.boolean().optional(),
+      profileChangedAt: Joi.number()
```

Comment:
nit: we should explicitly annotate this as `.optional()` for consistency with the other fields.